### DISCUSSION TITLE
[Feature]: Support for Olympus ORF raw format

### DISCUSSION THREAD

**Discussion by Sixth2538** [immich-app/immich#1653] [Feature Request]
### Feature detail

Currently ORF raw files are not being uploaded to the server. Will be great if this format can be supported (and maybe amongst other raw format if its not too much of an ask!)

### Platform

Server

**Comment by alextran1502** [comment]
Do you have a sample file that you can share for developing purpose?

**Comment by Sixth2538** [comment]
Sure thing.

Unable to upload .orf file directly, so uploaded to the following link:

---link removed---

Thank you for following up, will delete link once you have got a hand on the file.

Edit: Raw file provided by @ZeroEcks

**Comment by ZeroEcks** [comment]
FYI if you need sample raws from various cameras in the future, pixls.us is a valuable resource.

https://raw.pixls.us/

**Comment by Sixth2538** [comment]
Hi @alextran1502 would like to see if there are any updates on this issue. Thank you as always.

**Comment by alextran1502** [comment]
@Sixth2538 Not yet, I will add them along with additional raw format. Thank you for the ping!

**Comment by Sixth2538** [comment]
Thanks, looking forward to fully uploading my entire library to immich!

**Comment by uhthomas** [comment]
Resolved by #2834.
